src/models/file.js

```diff
@@ -21,13 +21,14 @@
  */
 import { getDavPath } from '../utils/fileUtils'
 
-export default function(fileInfo, mime, component) {
+export default function(fileInfo, mime, component, caplangs) {
```

Comment:
Hmmm, I'm not sure that caplangs should go here in `file.js` it is video specific. This change should not reach into this file.